### PR TITLE
JNG-4619 minor jsl fixes

### PR DESCRIPTION
**Pull request by gprivi**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-4619" title="JNG-4619" target="_blank">StoryJNG-4619</a>  Minor JSL fixes
  </td></table>
  <br />
 

JNG-4619 minor jsl fixes

### CODE REVIEW

**Review by bencegelei — APPROVED** [review]

**Review by noherczeg — APPROVED** [review]

**Review by robertcsakany — APPROVED** [review]

**Review by Madmat8 — APPROVED** [review]

File: model-test/src/test/java/hu/blackbelt/judo/meta/jsl/runtime/EnumDeclarationTests.xtend [hunk 1]
```diff
@@ -132,9 +132,7 @@ class EnumDeclarationTests {
             enum TestLiteral {
                 Aaa01 = 1;
                 Bbb02 = 2;
```
Comment (fmagnucz) [inline model-test/src/test/java/hu/blackbelt/judo/meta/jsl/runtime/EnumDeclarationTests.xtend:134]: Also add this enum literal: Ccc03 = 9999

File: model-test/src/test/java/hu/blackbelt/judo/meta/jsl/runtime/EnumDeclarationTests.xtend [hunk 2]
```diff
@@ -181,7 +192,6 @@ class EnumDeclarationTests {
...
```
Comment (fmagnucz) [inline model-test/src/test/java/hu/blackbelt/judo/meta/jsl/runtime/EnumDeclarationTests.xtend:195]: Also add this enum literal: Ccc03 = 9999

File: model-test/src/test/java/hu/blackbelt/judo/meta/jsl/runtime/NamedTests.xtend [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import hu.blackbelt.judo.meta.jsl.jsldsl.JsldslPackage
+import hu.blackbelt.judo.meta.jsl.validation.JslDslValidator
+
+@ExtendWith(InjectionExtension)
+@InjectWith(JslDslInjectorProvider)
+class NamedTests {
+    @Inject extension ParseHelper<ModelDeclaration>
+    @Inject extension ValidationTestHelper
+    
+	@Test
```
Comment (fmagnucz) [inline model-test/src/test/java/hu/blackbelt/judo/meta/jsl/runtime/NamedTests.xtend:20]: Use space characters insted of TAB, please

Comment (fmagnucz) [reply model-test/src/test/java/hu/blackbelt/judo/meta/jsl/runtime/NamedTests.xtend:20]: Please add the `@Requirement` annotation with the tested requirement ids.

**Review by fmagnucz — COMMENTED** [review]
The naming section isn't modified but the naming rules are modified. Is it intentional?

**Pull request merged**